packages/wallet/package.json

```diff
@@ -63,7 +63,7 @@
     "lodash": "^4.17.10",
     "magmo-wallet-client": "0.0.3",
     "nanoid": "^1.0.3",
-    "nitro-protocol": "https://github.com/statechannels/nitro-protocol.git#ag-include-contracts",
+    "nitro-protocol": "https://github.com/statechannels/nitro-protocol.git#expose_createRefuteTransaction",
```

Comment:
Note that I had to update the branch being referenced here as the `createRefuteTransaction` wasn't being exposed through the `transactions` export.

Reply:
Hopefully we can get this switched to the mono repo soon